wiki/Tournaments/oPHN/en.md

```diff
@@ -3,4 +3,10 @@
 Index page for all of the osu! Philippines Nationals series.
 
 - [osu! Philippines Nationals 2019 - osu!mania 4K](2019_4K)
+- [osu! Philippines Nationals 2022 - osu!catch](2022_catch)
+- [osu! Philippines Nationals 2022 - osu!mania](2022_mania)
+- [osu! Philippines Nationals 2022 - osu!taiko](2022_taiko)
+- [osu! Philippines Nationals 2023 - osu!](2023_osu)
+- [osu! Philippines Nationals 2023 - osu!mania](2023_mania)
+- [osu! Philippines Nationals 2023 - osu!taiko](2023_taiko)
 - [osu! Philippines Nationals 2023 - osu!catch](2023_catch)
```

Comment:
check titles here against the articles, as some don't match and some needed to be corrected in the articles